.github/workflows/announce-a-release.yml

```diff
@@ -0,0 +1,61 @@
+name: Announce a release
+
+on:
+  push:
+    tags: announce-\d+.\d+.\d+
+
+jobs:
+  announce:
+    name: Announcements
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout main
+        uses: actions/checkout@v2
+        with:
+          ref: "main"
+          token: ${{ secrets.RELEASE_TOKEN }}
+      - name: Release notes
+        uses: ponylang/release-bot-action@0.6.1
+        with:
+          entrypoint: publish-release-notes-to-github
+        env:
+          RELEASE_TOKEN: ${{ secrets.RELEASE_TOKEN }}
+      - name: Zulip
+        uses: ponylang/release-bot-action@0.6.1
+        with:
+          entrypoint: send-announcement-to-pony-zulip
+        env:
+          ZULIP_API_KEY: ${{ secrets.ZULIP_RELEASE_API_KEY }}
+          ZULIP_EMAIL: ${{ secrets.ZULIP_RELEASE_EMAIL }}
```

Comment:
There's instructions for how to create and get these in the release-bot-action README.

https://github.com/ponylang/release-bot-action#send-announcement-to-pony-zulip